pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -142,7 +142,7 @@ func (cp *capacityPlugin) OnSessionOpen(ssn *framework.Session) {
 		}
 
 		if job.PodGroup.Status.Phase == scheduling.PodGroupInqueue {
-			attr.inqueue.Add(job.GetMinResources())
+			attr.inqueue.Add(job.DeductSchGatedResources(job.GetMinResources()))
```

Comment:
I think its better to do the calcalation in GetMinResources function. Let GetMinResources return the min resource to run the job.